framework/test/test_verify_bugs.sh

```diff
@@ -93,9 +93,9 @@ work_dir="$test_dir/$PID"
 # Clean working directory
 rm -rf $work_dir
 for bid in $(echo $BUGS); do
-    # Skip all bug ids that do not exist in the commit-db
-    if ! grep -q "^$bid," "$BASE_DIR/framework/projects/$PID/commit-db"; then
-        warn "Skipping bug ID that is not listed in commit-db: $PID-$bid"
+    # Skip all bug ids that do not exist in the active-bugs csv
+    if ! grep -q "^$bid," "$BASE_DIR/framework/projects/$PID/$BUGS_CSV_ACTIVE"; then
+        warn "Skipping bug ID that is not listed in active-bugs csv: $PID-$bid"
```

Comment:
Seems unnecessary, given that the bug ids are extracted from the same file on line 71. Delete this check?

Reply:
This check is needed if the user specified a list of bugs that don't exist. Line 71 is only run if the user does not already specify a list of bugs.